docs/intentReaderHandlers/RegisterProviderHandler.md

```diff
@@ -0,0 +1,108 @@
+# RegisterProviderHandler 
+
+## Overview
+
+RegisterProviderHandler is invoked when the task specified in the intent has the value "registerProviderHandler". This handler is used to trigger the provider and is invoked once an entity provides services to another/external party.
+
+It performs the following actions :
+1. Parse the input message received to get the provider.
+2. Trigger the provider passed in params.
+3. Save the response/error.
+4. Format the result and send the response back to the IntentReader.
```

Comment:
Same comment left on other docs can we do
Parses
Triggers the provider passed in the params.
Saves
Formats the result and sends

Reply:
updated for all handlers